[🐛] saveToPhotos causes app to crash on Android 11 when targetSdkVersion is set to 30

### Description

I have an issue, where on Android 11 devices using `saveToPhotos: true` camera option will crash the application upon confirming taken photo, when `targetSdkVersion` is set to 30.

Seems it tries to save the taken picture to the media library using MIME type `application/octet-stream`.

### How to repeat issue and example

- Use Android 11 device
- In `build.gradle` file, set `targetSdkVersion` and `compileSdkVersion` to `30`
- Take a picture using `launchCamera` -method with `saveToPhotos` camera option set to `true`
- Confirm taken picture

### Additional Information

- Image Picker version: 3.6.0
- React Native version: 0.64.0
- Platform:  Android
- Development Operating System: Windows 10
- Dev tools: Android Studio version: 4.1.2, Android SDK version: 30

### Error log

```
Caused by: java.lang.IllegalArgumentException: MIME type application/octet-stream cannot be inserted into content://media/external/images/media; expected MIME type under image/*

FATAL EXCEPTION: main
Process: PACKAGE_NAME, PID: XXXX
java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=13001, result=-1, data=Intent {  }} to activity {PACKAGE_NAME/PACKAGE_NAME.MainActivity}: java.lang.IllegalArgumentException: MIME type application/octet-stream cannot be inserted into content://media/external/images/media; expected MIME type under image/*
	at android.app.ActivityThread.deliverResults(ActivityThread.java:5015)
	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5056)
	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:51)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7656)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: java.lang.IllegalArgumentException: MIME type application/octet-stream cannot be inserted into content://media/external/images/media; expected MIME type under image/*
	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:172)
	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:142)
	at android.content.ContentProviderProxy.insert(ContentProviderNative.java:549)
	at android.content.ContentResolver.insert(ContentResolver.java:2149)
	at android.content.ContentResolver.insert(ContentResolver.java:2111)
	at com.imagepicker.Utils.saveToPublicDirectory(Utils.java:83)
	at com.imagepicker.ImagePickerModule.onActivityResult(ImagePickerModule.java:171)
	at com.facebook.react.bridge.ReactContext.onActivityResult(ReactContext.java:305)
	at com.facebook.react.ReactInstanceManager.onActivityResult(ReactInstanceManager.java:762)
	at com.facebook.react.ReactDelegate.onActivityResult(ReactDelegate.java:90)
	at com.facebook.react.ReactActivityDelegate.onActivityResult(ReactActivityDelegate.java:112)
	at com.facebook.react.ReactActivity.onActivityResult(ReactActivity.java:68)
	at android.app.Activity.dispatchActivityResult(Activity.java:8310)
	at android.app.ActivityThread.deliverResults(ActivityThread.java:5008)
	... 11 more
```

--- a/android/src/main/java/com/imagepicker/Utils.java
+++ b/android/src/main/java/com/imagepicker/Utils.java
@@ -80,9 +80,11 @@ public class Utils {
 
         if (mediaType.equals("video")) {
             fileDetails.put(MediaStore.Video.Media.DISPLAY_NAME, UUID.randomUUID().toString());
+            fileDetails.put(MediaStore.Video.Media.MIME_TYPE, resolver.getType(uri));
             mediaStoreUri = resolver.insert(MediaStore.Video.Media.EXTERNAL_CONTENT_URI, fileDetails);
         } else {
             fileDetails.put(MediaStore.Images.Media.DISPLAY_NAME, UUID.randomUUID().toString());
+            fileDetails.put(MediaStore.Images.Media.MIME_TYPE, resolver.getType(uri));
             mediaStoreUri = resolver.insert(MediaStore.Images.Media.EXTERNAL_CONTENT_URI, fileDetails);
         }
 

--- a/example/src/App.js
+++ b/example/src/App.js
@@ -23,6 +23,7 @@ export default function App() {
           onPress={() =>
             ImagePicker.launchCamera(
               {
+                saveToPhotos: true,
                 mediaType: 'photo',
                 includeBase64: false,
                 maxHeight: 200,
@@ -64,7 +65,7 @@ export default function App() {
         <Button
           title="Select video"
           onPress={() =>
-            ImagePicker.launchImageLibrary({mediaType: 'video'}, (response) => {
+            ImagePicker.launchImageLibrary({mediaType: 'video', saveToPhotos: true}, (response) => {
               setResponse(response);
             })
           }

